### PR TITLE
Update documentation for Proxy IP addresses and ports

### PR DESCRIPTION
**Pull request by asaj**
### Description

This PR explains which ports need to be open on which IPs, and adds commands to check that the machine is accepting TCP connections over those ports.

Does not check UDP.

### Tested

Not tested

### Other changes

None

### CODE REVIEW

**Comment by asaj** [comment]
@nambrot please let me know what I'm missing for the attestations service

**Review by lucasege — APPROVED** [review]

**Comment by nambrot** [comment]
For the attestation machine, it would be port 30303 as well on udp/tcp for discovery of the node, and then port 80 for the service itself

File: packages/docs/getting-started/running-a-validator.md [hunk 1]
```diff
@@ -91,26 +105,27 @@ Note that account and signer keys must be unique and may not be reused.
...
+| CELO_VALIDATOR_SIGNER_SIGNATURE      | The proof-of-possession of the validator signer key                                                                                  |  |
+| CELO_VALIDATOR_SIGNER_BLS_PUBLIC_KEY | The BLS public key for the Validator instance                                                                                        |  |
+| CELO_VALIDATOR_SIGNER_BLS_SIGNATURE  | A proof-of-possession of the BLS public key                                                                                          |  |
+| PROXY_ENODE                          | The enode address for the Validator proxy                                                                                            |  |
+| PROXY_INTERNAL_IP                    | (Optional) The internal IP address over which your Validator can communicate with your Proxy                                         |  |
+| PROXY_EXTERNAL_IP                    | The external IP address of the Proxy. May be used by the Validator to communicate with the Proxy if PROXY_INTERNAL_IP is unspecified |  |
```
Comment (marekolszewski) [inline packages/docs/getting-started/running-a-validator.md:121]: Maybe we can add that people can test their external IP address by running the following command:

`dig +short myip.opendns.com @resolver1.opendns.com`

Comment (asaj) [reply packages/docs/getting-started/running-a-validator.md:121]: There are instructions below to use telnet, is `dig` preferable?

Comment (marekolszewski) [reply packages/docs/getting-started/running-a-validator.md:121]: Telnet is good at checking whether your port is open, but this command tells you what your external IP is.

File: packages/docs/getting-started/running-a-validator.md [hunk 2]
```diff
@@ -253,11 +268,13 @@ echo $(docker exec celo-proxy geth --exec "admin.nodeInfo['enode'].split('//')[1
...
 
 Now we need to set the proxy enode and proxy IP address in environment variables on the validator machine.
+If you don't have an internal IP address over which the Validator and Proxy can communicate, feel free to set the internal IP address to the external IP address.
 
 ```bash
 # On the validator machine
 export PROXY_ENODE=<YOUR-PROXY-ENODE>
-export PROXY_IP=<PROXY-MACHINE-EXTERNAL-IP-ADDRESS>
+export PROXY_EXTERNAL_IP=<PROXY-MACHINE-EXTERNAL-IP-ADDRESS>
+export PROXY_INTERNAL_IP=<PROXY-MACHINE-INTERNAL-IP-ADDRESS>
```
Comment (marekolszewski) [inline packages/docs/getting-started/running-a-validator.md:286]: Maybe we can list that dig command here instead.

**Review by marekolszewski — APPROVED** [review]
Looks good! Again I would add that dig command before merging.

File: packages/docs/getting-started/running-a-validator.md (hunk 1 above)
Comment (asaj) [reply packages/docs/getting-started/running-a-validator.md:121]: Ah gotcha, perfect, thanks.

**Pull request merged**